src/source-af-packet.c

```diff
@@ -782,6 +782,7 @@ int AFPReadFromRing(AFPThreadVars *ptv)
             SCReturnInt(AFP_FAILURE);
         }
         PKT_SET_SRC(p, PKT_SRC_WIRE);
+        p->pkt_mode = PKT_MODE_AUTO;
```

Comment:
I don't think it works. StreamTcpInlineMode() checks explicitly for IDS or IPS, or "return (stream_inline && PacketModeIsIPS(p));" PacketModeIsIPS again, checks for IPS. So where does 'AUTO' get handled?